### ISSUE TITLE
Release 3.5.0-beta - Config-File-Format-Change - Release Notes ? 

### ISSUE THREAD

**Issue by stefanCCS** [kitodo/kitodo-production#5443]
It looks like, that the provided config files are in "Unix" style now and not anymore in "Windows" style (I refer to End-of-line convention).
(e.g. I have made some checks for config files in WAR-File WEB-INF/classes).
Which I like very much :-) and I want to keep it in "Unix" style.
Anyway, a hint in the Release Notes would be good, like "changed files to 'Unix' style...".
(e.g. I struggled a bit about the fact, that a "diff" between these new files and the older ones gives a lot of hits).

**Comment by solth** [comment]
That's interesting. I still had to run the command `sudo dos2unix scripts/*.sh` from the installation guide to make it work.

**Comment by henning-gerhardt** [comment]
Keep in mind that the line endings depends on the system where the checkout is made and the release is created as in the `.gitattributes` file the line endings are set on `auto` for text files - only for `.bat` files they are set to Windows and on `.sh` files on Unix line endings. If the checkout and release is created on Windows then you get Windows line endings in any text file inside the release, on Unix/Linux you get Unix/Linux line endings, on MacOS you get the MacOS line ending.

**Comment by henning-gerhardt** [comment]
The provided files in  `kitodo_3-5-0-beta-1_config_modules.zip ` are in most cases with Window line endings and only a few with Unix/Linux line endings. This is the reason why running `dos2unix *.sh` is still necessary on Unix/Linux to get the `.sh` files work correct. In the source code repo the files are correct according to the used operation system, if the checkout was made in a correct way.

**Comment by henning-gerhardt** [comment]
@stefanCCS : running diff command with option -w is recommend as this parameter tells diff that any white space including line ending differences should be ignored.

**Comment by stefanCCS** [comment]
Just for understanding:
I only have made a simple download of the released files - here the [WAR-File](https://github.com/kitodo/kitodo-production/releases/download/kitodo-production-3.5.0-beta-1/kitodo-3.5.0-beta-1.war).
Then, I have extracted it (using 7-ZIP on Windows), and checked the config files in WEB-INF/classes.
-> These were in UNIX-Style (and have been in Windows Style in former Releases).

Thanks for the hint with the `diff` - (I have found out, that `--strip-trailing-cr` also works fine in this case).

Anyway, I prefer to have all files consistently in Unix-Style.

If this is not easy to to change (or even to check), my proposal still is to give a small hint in the Release-Notes, that these files might be Unix- or Windows-Style and depending on installation, they need to be converted.
